docs/admin/prometheus.md

```diff
@@ -25,6 +25,19 @@ coderd_api_active_users_duration_hour 0
 The Prometheus endpoint can be enabled in the [Helm chart's](https://github.com/coder/coder/tree/main/helm) `values.yml` by setting the environment variable `CODER_PROMETHEUS_ADDRESS` to `0.0.0.0:2112`.
 The environment variable `CODER_PROMETHEUS_ENABLE` will be enabled automatically.
 
+### Prometheus configuration
+
+To allow Prometheus to scrape the Coder metrics, you will need to create a `scape_config`
+in your `prometheus.yml` file, or in the Prometheus Helm chart values. Below is an
+example `scrape_config`:
+
+```yaml
+scrape_configs:
+  - job_name: "coder"
+    static_configs:
+      - targets: ["<ip>:2112"] # replcace with the the IP address of the Coder pod or server
```

Comment:
I think we should also mention that the prom server needs to be enabled on coder as well.

```suggestion
  - job_name: "coder"
    scheme: "http"
    static_configs:
      - targets: ["<ip>:2112"] # replace with the the IP address of the Coder pod or server
        labels:
          app: "coder"
```